src/app/components/NavBar/NavBar.module.css

```diff
@@ -2,12 +2,11 @@
   margin: 0;
   background-color: var(--primary-color);
   display: grid;
-  grid-template-rows: auto auto;
+  grid-auto-flow: row;
```

Comment:
Is the `grid-auto-flow` `row` the default?
https://developer.mozilla.org/en-US/docs/Web/CSS/grid-auto-flow#formal_definition